java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -225,4 +225,36 @@ public interface ListBaseCommands {
      * }</pre>
      */
     CompletableFuture<String[]> rpopCount(String key, long count);
+
+    /**
+     * Inserts specified values at the tail of the <code>list</code>, only if <code>key</code> already
+     * exists and holds a list.
+     *
+     * @see <a href="https://redis.io/commands/rpushx/">redis.io</a> for details.
+     * @param key The key of the list.
+     * @param elements The elements to insert at the tail of the list stored at <code>key</code>.
+     * @return The length of the list after the push operation.
+     * @example
+     *     <pre>{@code
+     * Long pushCount1 = client.rpushx("my_list", new String[] {"value1", "value2"}).get();
+     * assert pushCount1 == 2L;
```

Comment:
Can we rename the variable to clarify that the returned value is the new length of the list instead of the number of added elements? Also, does Redis allow storing empty lists? If not, the new length has to be greater than 2 since rpushx only operates on existing lists. If you can store empty lists than the return value of 2 makes sense. Same applies to the lpushx and other rpushx docs in this PR
```suggestion
     * Long listLength = client.rpushx("my_list", new String[] {"value1", "value2"}).get();
     * assert listLength > 2L;
```

Reply:
Thank you, fixed